components/threading/thread_viewer/thread_viewer.tsx

```diff
@@ -311,15 +311,15 @@ export default class ThreadViewer extends React.Component<Props, State> {
     // scrolls to either bottom or new messages line
     private onInit = async (reconnected = false): Promise<void> => {
         if (reconnected || this.morePostsToFetch()) {
-            this.setState({isLoading: true});
+            this.setState({isLoading: !reconnected});
```

Comment:
Is there any case where both are true? (and therefore, it should show the loader?

Reply:
A little background: `onInit` gets called on 3 occasions: mount / `selectedTheadChanged` / reconnect.
It should show the loader at mount and `selectedThreadChanged`, and only if the thread is not wholly loaded already. 
So it would be `true` everytime it gets to be set with the exception of re-connection.